[BALANCE] Double Roundstart TC's from 20 to 40.

Steelpoint opened the pull request:
In response to the discussions on the forums, this PR is intended as a first, small, step forward in effecting some change to the game.

https://tgstation13.org/phpBB/viewtopic.php?f=10&t=3802 (How Lethal Should SS13 Be?)
https://tgstation13.org/phpBB/viewtopic.php?f=9&t=3556 (Migrants)
https://tgstation13.org/phpBB/viewtopic.php?f=10&t=3799 ("Removing People From The Round")
https://tgstation13.org/phpBB/viewtopic.php?f=9&t=3781 (Respawn Mechanics)

This change is simple, Traitors and DA's will start the game with 40 Telecrystals instead of 20.

No other values are changed, at this time.

Steelpoint committed:
40TCs

HoaxIT commented:
:+1: SS13 should be fukken lethal, git gud and die often.

Ausops commented:
How about telecrystals being scaled to population or sec+silicon population.

Durkel commented:
Just doubling the amount of tcs traitors get is a lazy way to solve a issue and it doesn't compensate changelings or another antag.

freerealestate commented:
I'm not sure that DAs should start with double TC's. Everyone always complains about what a clusterfuck those rounds are. It's the regular traitor rounds that need buffing.

Steelpoint commented:
I'll look into small balancing of some traitor gear to compensate but I want to be wary of going in to heavily initially. Little point of doing this PR if I just also double the cost of all traitor items.

Dorsisdwarf commented:
I agree that DA should only get 20 TC or so, to lessen the double-exclamationmark-Fun-double-exclamationmark

GunHog commented:
Make it a scale. The issue antagonists have is population. Increased assistant/engineer activity in maint, increased likelihood that a target will have buddies, higher security officer and cyborg counts are all major factors that work against an antagonist. However, in low-population, there is a chance that there will be no AI player at all, Security may only consist of one or two players, and there may not even be a Captain or HoS player. In those situations, antagonists, even as they are now, are hilariously overpowered.

A scaling TC count would help to resolve both of these perceived issues with traitors.

Steelpoint commented:
Traitor and DA Uplinks appear to be tied to the same thing, I'll do a one over the individual items however.

Steelpoint committed:
MinorChanges

Steelpoint commented:
Changed the following prices

Syndicate Pistol to 10 from 9
Revolver to 15 from 13
Esword to 10 from 8
Minibomb to 8 from 6

Revolver Ammo to 6 from 4

Chameleon Jumpsuit to 1 from 4
Agent Card to 5 from 3

Op Medkit to 5 from 9

Hacked AI Module to 30 from 14
Sing Beacon to 30 from 14
Syndicate Bomb to 15 from 11

Adrenal Implant to 10 from 8
Freedom Implant to 8 from 5
Uplink Implant remains as is, store half your TC's for later.

Balloon to 40 from 20
Surplus Crate to 40 from 20

freerealestate commented:
>A scaling TC count would help to resolve both of these perceived issues
with traitors.

Good idea

On 11 June 2015 at 02:00, Steelpoint <notifications@github.com> wrote:

> Changed the following prices
>
> Syndicate Pistol to 10 from 9
> Revolver to 15 from 13
> Esword to 10 from 8
> Minibomb to 8 from 6
>
> Revolver Ammo to 6 from 4
>
> Chameleon Jumpsuit to 1 from 4
> Agent Card to 5 from 3
>
> Op Medkit to 5 from 9
>
> Hacked AI Module to 30 from 14
> Sing Beacon to 30 from 14
> Syndicate Bomb to 15 from 11
>
> Adrenal Implant to 10 from 8
> Freedom Implant to 8 from 5
> Uplink Implant remains as is, store half your TC's for later.
>
> Balloon to 40 from 20
> Surplus Crate to 40 from 20
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/9881#issuecomment-110765270>
> .
>

Miauw62 commented:
>double

doubling this is a very big change. maybe do something like 30 TC?

Cheridan commented:
I don't see this helping anything.

>Just doubling the amount of tcs traitors get is a lazy way to solve a issue and it doesn't compensate changelings or another antag.

All this does is give traitors more points to spend on bombs so they can discreetly end the round. If we want traitors to be strong again we have to work from the other direction.

Steelpoint commented:
I already slightly buffed the cost of bombs and can do so further.

Right now you can..
 * Buy two Syndi Bombs, up from one. Leaving you with 10 TC's
*  Buy five Minibombs, up from three. Leaving you with no TC's.

As I said the final number is still up for debate.

Steelpoint commented:
One more point.

I agree this is not the best solution, and that anything we're going to do to solve the issue of boring game play will have to occur over a long period of time with many PR's and fixes. 

But this is a good first step in the right direction, its quick, easy and can be easily reverted or altered.

Miauw62 commented:
also please buff surplus crates accordingly.

Steelpoint commented:
I did, the surplus crate and the balloon have both been updated to cost 40 TCs.

Cookiesndip commented:
> Traitors AND DAs having TC doubled.

I feel like DAs should stay at 20, yet have 1.5-2x the number of traitors (quantity over quality), while traitors should receive the buff in full. Otherwise security are going to be fucked at roundstart dealing with 15~ antags with 40 TC available to each. I also feel that there should be stock limitations for low-effort, mass-destructive items (per person), i.e. syndicate bombs and power sinks, since they aren't constrained by TC any longer.

On the other hand, more chaos and less control is fantastic at the moment, so I don't really know where I stand here.

Miauw62 commented:
i meant the yield of the surplus crate, steelpoint.

Steelpoint commented:
The crate should be good but I'll triple check.

Changing values around some more, reverting most of the changes I've done but I'm further increasing the price of explosives so you can't buy more than you can right now.

Minibombs - 12 TC's
Syndi Bomb - 25 TC's

Steelpoint committed:
Bombs

tedward1337 commented:
Y E S

Xhuis commented:
I N F L A T I O N

tedward1337 commented:
I NEED THIS IN MY SPACEMENS

HoaxIT commented:
So, how will this affect `nuke ops` and `double agents`

Steelpoint commented:
Double Agents will become a clusterfuck and Nuke Ops are unaffected as they have their own individual uplink that gets 20 TC's plus scaling TC's.

I'm not going to touch Nuke Ops at this time as doing so will require more in depth balancing of their guns and equipment.

Jalleo commented:
I really dont like this due to the fact that this is now becoming a creep in silly ways. We really should be more bundling together items that are useful together to help keep choices simple while keeping certain items which are a important decision hard to easily choose on.

Secondly we should have a mechanic to earn new TC not just flat out have more TC

more starting TC not equal more deaths. It could end up with security just getting more gear by having spare TC left by dead traitors.

SconesC commented:
i hope you're ready for:
- ebow
- esword
- noslips
- thermals
- adrenal
- c4

in one bangup maintstalker loadout

HoaxIT commented:
@Steelpoint You're increasing items of several traitor items, this will affect nuke ops.

KorPhaeron commented:
What Scones said. Hell just being able to buy a revolver, adrenal, and pile of ammo will result in a blood bath

Steelpoint commented:
Not necessarily a bad thing.

I'll be interested in seeing how a doubling of TC's affects the game, and as noted it will be real easy to quickly change the values around to address serious issues.

SconesC commented:
I'm just gonna buy that every round and kill people, 40 TC is too much and like I said in the thread we will never fix lethality without changing core systems

KorPhaeron commented:
Please at least do scaling. I know nobody gives a shit about basil but 40 TC with 13-20 people would result in a dead station

Chocobro commented:
i'm kind of not a fan
but i haven't played an antag role in over a year.

Aranclanos commented:
WAKE ME UP

tedward1337 commented:
(CAN'T WAKE UP)

Dorsisdwarf commented:
I like that surplus crates now contain 100 TC of kit

Fox-McCloud commented:
Not a fan, either; think this would be better to slowly wratchet up; start by increasing the TC by like 5-10 and not changing the prices of anything first---then make further tweaks, IMO.

Doubling it, even with adjusted costs is going to be out and out ridiculous in a lot of scenarios.

PKPenguin321 commented:
\>40 soaps

merge it

PKPenguin321 commented:
but honestly I know this may be the unpopular opinion but if this was ran for a week or two i would really like to see how it plays out, :+1: from me

kyrahabattoir commented:
:thumbsdown:

ike709 commented:
:-1: 20 TC plus highpop scaling please

Iamgoofball commented:
>Chameleon Jumpsuit to 1 from 4

are you fucking kidding me

this is already underused why are you making it even less used

Tokiko1 commented:
Don't forget to adjust the amount of TC a surplus crate contains.

Lumipharon commented:
Jjust double tc's, double all item costs, THEN adjust UNDERUSED item's cost down.

All this pr does is let every traitor be turbomaintmurderboner50000.

What we DO need, is more traitors. Seriously, we have rounds with 20+ people in them, and like 3 traitors.

Xhuis commented:
Likely conflicts with #9789

Steelpoint commented:
@Iamgoofball Firstly it says I adjusted the price to one telecrystal, where it was originally four telecrystals

Secondly I reverted that change to focus only on ensuring you can't buy more explosives than you can already.

Fox-McCloud commented on code/datums/uplink_item.dm:
```diff
@@ -714,7 +714,7 @@ var/list/uplink_items = list()
 	U.uses -= cost
-	U.used_TC = 20
+	U.used_TC = 40
 	var/remaining_TC = 50
```
this needs to be 100 now.

Previously you paid 20 and got 50 TC---meaning effectively x2.5 TC; if you're really doubling TC, then paying 40 to get an extra free 10 TC will almost never be worth it.

Cheridan commented:
I'm closing this. If there's a particular balance issue you feel needs addressed with the cost of items then you can change the cost, but just throwing telecrystals at the problem is aimless and I doubt it'll actually help anything (oh boy, I can get forty soaps or ineffectively dual-wield ebows).

We don't need *sweeping* changes to what people often refer to as our only good game mode.
